cmd/exec.go

```diff
@@ -72,7 +72,7 @@ func executeExec(ctx context.Context, dev *model.Dev, args []string) error {
 		dev.Namespace = namespace
 	}
 
-	p, err := pods.GetDevPod(ctx, dev, client, false)
+	p, err := pods.GetDevPod(ctx, dev, client, false, nil)
```

Comment:
I think calling this `GetDevPod` is confusing, since it calls and monitors. Maybe we can have two functions, one to get the pod and another to start the watcher?